changelog/13115.improvement.rst

```diff
@@ -0,0 +1 @@
+Allows supplying ``ExceptionGroup[Exception]`` and ``BaseExceptionGroup[BaseException]`` to ``pytest.raises`` to keep full typing on ExcInfo.
```

Comment:
I'd explicitly say "Parametrizing with other element types remains an error - we do not check the types of child exceptions and thus do not permit code that might look like we do."

Reply:
Also added in latest commit